basic-conditionals.tex

```diff
@@ -0,0 +1,400 @@
+\begin{itemize}
+  \item \verb+==+: primerja dve številski vrednosti. Rezultat je pravilen, če sta vrednosti enaki.
+  \item \verb+!=+: primerja dve številski vrednosti. Rezultat je pravilen, če sta vrednosti različni.
+  \item \verb+<+: primerja dve številski vrednosti. Rezultat je pravilen, če je vrednost na levi manjša od vrednosti na desni.
+  \item \verb+>+: deluje podobno kot \verb+<+, le da v drugo smer; rezultat je pravilen, če je vrednost na desni manjša od vrednosti na levi.
+  \item \verb+<=+: primerja dve številski vrednosti. Rezultat je pravilen, če sta vrednosti enaki, ali če je vrednost na levi manjša od vrednosti na desni.
+  \item \verb+>=+ deluje podobno kot \verb+<=+, le da v drugo smer.
+\end{itemize}
+
+
+\begin{examples}
+
+Poglejmo si primer uporabe pogojnega stavka.
+
+\begin{Verbatim}[commandchars=\\\{\}]
+\PY{c+cp}{\PYZsh{}}\PY{c+cp}{include} \PY{c+cpf}{\PYZlt{}stdio.h\PYZgt{}}
+
+\PY{k+kt}{int} \PY{n+nf}{main}\PY{p}{(}\PY{p}{)} \PY{p}{\PYZob{}}
+    \PY{k+kt}{int} \PY{n}{a}\PY{p}{,} \PY{n}{b}\PY{p}{;}
+    \PY{n}{scanf}\PY{p}{(}\PY{l+s}{\PYZdq{}}\PY{l+s}{\PYZpc{}d\PYZpc{}d}\PY{l+s}{\PYZdq{}}\PY{p}{,} \PY{o}{\PYZam{}}\PY{n}{a}\PY{p}{,} \PY{o}{\PYZam{}}\PY{n}{b}\PY{p}{)}\PY{p}{;}
+        
+    \PY{k}{if} \PY{p}{(}\PY{n}{a} \PY{o}{=}\PY{o}{=} \PY{n}{b}\PY{p}{)} \PY{p}{\PYZob{}}
+        \PY{n}{printf}\PY{p}{(}\PY{l+s}{\PYZdq{}}\PY{l+s}{Stevilki sta enaki.}\PY{l+s+se}{\PYZbs{}n}\PY{l+s}{\PYZdq{}}\PY{p}{)}\PY{p}{;}
+    \PY{p}{\PYZcb{}}
+    \PY{k}{if} \PY{p}{(}\PY{n}{a} \PY{o}{!}\PY{o}{=} \PY{n}{b}\PY{p}{)} \PY{p}{\PYZob{}}
+        \PY{n}{printf}\PY{p}{(}\PY{l+s}{\PYZdq{}}\PY{l+s}{Stevilki sta razlicni.}\PY{l+s+se}{\PYZbs{}n}\PY{l+s}{\PYZdq{}}\PY{p}{)}\PY{p}{;}
+    \PY{p}{\PYZcb{}}
+    \PY{k}{if} \PY{p}{(}\PY{n}{a} \PY{o}{\PYZlt{}} \PY{n}{b}\PY{p}{)} \PY{p}{\PYZob{}}
+        \PY{n}{printf}\PY{p}{(}\PY{l+s}{\PYZdq{}}\PY{l+s}{Prva stevilka je manjsa od druge.}\PY{l+s+se}{\PYZbs{}n}\PY{l+s}{\PYZdq{}}\PY{p}{)}\PY{p}{;}
+    \PY{p}{\PYZcb{}}
+    \PY{k}{if} \PY{p}{(}\PY{n}{a} \PY{o}{\PYZgt{}} \PY{n}{b}\PY{p}{)} \PY{p}{\PYZob{}}
+        \PY{n}{printf}\PY{p}{(}\PY{l+s}{\PYZdq{}}\PY{l+s}{Prva stevilka je vecja od druge.}\PY{l+s+se}{\PYZbs{}n}\PY{l+s}{\PYZdq{}}\PY{p}{)}\PY{p}{;}
+    \PY{p}{\PYZcb{}}
+    \PY{k}{if} \PY{p}{(}\PY{n}{a} \PY{o}{\PYZlt{}}\PY{o}{=} \PY{n}{b}\PY{p}{)} \PY{p}{\PYZob{}}
+        \PY{n}{printf}\PY{p}{(}\PY{l+s}{\PYZdq{}}\PY{l+s}{Prva stevilka je manjsa ali enaka drugi.}\PY{l+s+se}{\PYZbs{}n}\PY{l+s}{\PYZdq{}}\PY{p}{)}\PY{p}{;}
+    \PY{p}{\PYZcb{}}
+    \PY{k}{if} \PY{p}{(}\PY{n}{a} \PY{o}{\PYZgt{}}\PY{o}{=} \PY{n}{b}\PY{p}{)} \PY{p}{\PYZob{}}
+        \PY{n}{printf}\PY{p}{(}\PY{l+s}{\PYZdq{}}\PY{l+s}{Prva stevilka je vecja ali enaka drugi.}\PY{l+s+se}{\PYZbs{}n}\PY{l+s}{\PYZdq{}}\PY{p}{)}\PY{p}{;}
+    \PY{p}{\PYZcb{}}
+    \PY{k}{return} \PY{l+m+mi}{0}\PY{p}{;}
+\PY{p}{\PYZcb{}}
+\end{Verbatim}
+
+\begin{inout}
+12 12
+\tcblower
+Stevilki sta enaki.\\
```

Comment:
`stevilo` je bolj povabljivo kot `stevilka`

Reply:
Ok, ampak zakaj že?

Reply:
https://fran.si/iskanje?FilteredDictionaryIds=130&View=1&Query=%C5%A1tevilka 

2. definicija pravi da je stevilka pogovorno za to kar hoces ti rect, mi pa hocmo da je to kokr tolk nepogovorno napisano